server-side/managers/sharingManager.js

```diff
@@ -0,0 +1,48 @@
+import Users from '../models/user.model';
+import PendingUsers from '../models/pendingUser.model';
+import { createPendingUser } from '../controllers/pendingUser.controler';
+import { createInvitation } from '../controllers/invitation.controler';
```

Comment:
You shouldn't call functions from the controller. On the server side, the controller is used for communication with the client.
you should move the logic to the service file, and use it here.
do it for both: createPendingUser, createInvitation